CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 # CHANGELOG
 
+## 8.0.1
+- Fix the exporting and importing of typings in ES6 build module
```

Comment:
```suggestion
- Fix the exporting and importing of typings in ES6 build module (#1164, thanks @diego-antonelli!)
```

I'm a firm believer in credit where it's due and you, sir, have earned internet points :-)